llvm/test/tools/llvm-cov/mcdc-maxbs.test

```diff
@@ -0,0 +1,21 @@
+# Test the assertion failure in llvm-cov with empty bitmap (#78922)
```

Comment:
Can you check for the specific assert?

Reply:
Passing w/o assertions is expected, so check(s) would not make sense, I think.
Let me update this sentence since it is misleading.